Revamped homepage

erciccione opened the pull request:
These changes aim to make the homepage a better source of info for the visitor. Right now the main section is "Why Monero is different", which is not something we should explain to somebody visiting the homepage (especially for the first time, different from what?). The point of these changes is to give immediately all the resources necessary to get started with Monero and, equally important, to easily find the info they might be looking for on getmonero.

I feel that the website is not seen as a point of reference and i think part of the problem is that it's not easy to find the info people need. This PR is part of the efforts of making the website easier to consult and the pages more interconnected.

The experience i aim to give is:

People open the website and they immediately see the promo video, which is a great first impact resource. If they know nothing about Monero, they will probaly click the button "What is Monero?" and they will be taken to the page where they can find more info.

If they are already somehow familiar with Monero, they will probably scroll down, where they will be able to directly find the info they are likely looking for: Download a wallet, get some coins, places where to spend their coins and a link to the FAQ, so if they have doubts, they can resolve them right away.

If they want more detailed info or are developers looking for instructions, we link to the guides and the library. If they are interested in contributing they can go to the "Contributing" page (which needs some big improvements).

In this PR i also removed the "Using" page, since those information are now in the homepage and i used the pictures in the new home. @binaryfate it will be probably necessary to create a redirect so that people visiting https://web.getmonero.org/get-started/using/ will be redirected to the homepage. I don't think many websites link to that page, but better not to risk to give a 404.

**Preview:**
![Screenshot_2020-04-22  The Monero Project ](https://user-images.githubusercontent.com/28106476/79961829-8bda1c80-8487-11ea-93d3-899f6a202a88.png)

**Before:**
![before](https://user-images.githubusercontent.com/28106476/82139717-43194780-982a-11ea-9977-126edaad3a0e.png)

SamsungGalaxyPlayer commented:
I think we need one tile the size of the "Guides and Resources" above the downloads to help convince newcomers a bit more. A few icons with the below descriptions:

| | | | |
| --- | --- | --- | --- |
| Icon: | ? | M = M coin icon for fungibility | Current "secure" |
| Text: | Privacy for all | Coin equality | Security |

erciccione commented:
@SamsungGalaxyPlayer I don't agree with putting more info about Monero in the homepage. I'm proposing these changes with the primary goal of removing the ones we had before :)

We currently have 2 pages ('About' and 'What is Monero?') and 2 generic videos where people can find general info about Monero, i think it's better to point to resources we already have (as i tried to do adding the button with the link ot the 'What is Monero' page).

A newcomer won't need to be convinced, if they arrived on getmonero without knowing nothing about Monero, they are immediately shown the promo video and the short introduction text with Link to the "What is Monero" page, which contains more detailed information, and right after they will find all the instructions to immediately get started and the FAQ.

This change aims to make the homepage more like a hub of resources. If we want to add sligtly more info, we could edit or slightly expand the text of the first block, but that could be done in another PR. We could also improve the "What is Monero" page, which is quite old (this is in my todo list already).

SarangNoether commented on _i18n/en.yml:
```diff
@@ -54,36 +54,33 @@ index:
-  monero_is_fungible_para2: fungible
-  monero_is_fungible_para3: because it is private by default. Units of Monero cannot be blacklisted by vendors or exchanges due to their association in previous transactions.
+  choosewallet: Choose a Wallet
+  choosewallet_para: To use Monero, first thing you are going to need is a wallet. Visit our Downloads page and get the right wallet for you. The Monero wallets are available for a variety of platforms and contain everything you need to use Monero immediately  and, if you want, run your own node.
```
s/first thing/the first thing/
s/immediately  and/immediately and/

erciccione replied:
` s/immediately and/immediately and/` What's the difference here?

SarangNoether replied:
Appears to be an extra space (that Markdown removed in my comment).

erciccione replied:
I see it in the source now. Will fix with the next push.

SarangNoether commented on _i18n/en.yml:
```diff
@@ -54,36 +54,33 @@ index:
-  latest_news: Latest News
-  more_news: More news
+  getcoins: Get Some Coins
+  getcoins_para: After you installed a wallet, you need to get some Monero. There are multiple ways to acquire some coins to spend, like mining or working in exchange for Monero, but the easiest way is to use an exchange and convert your fiat money into XMR. any exchanges, centralized and decentralized, list Monero (XMR).
```
s/installed/install/
s/XMR. any/XMR. Many/

SarangNoether commented on _i18n/en.yml:
```diff
@@ -54,36 +54,33 @@ index:
+  getcoins_para: After you installed a wallet, you need to get some Monero. There are multiple ways to acquire some coins to spend, like mining or working in exchange for Monero, but the easiest way is to use an exchange and convert your fiat money into XMR. any exchanges, centralized and decentralized, list Monero (XMR).
+  exchanges: Exchanges
+  useit: "It's a currency: Use it!"
+  useit_para: Monero is a currency and can be exchanged for goods and services, privately and with very low fees. Many commerciants will gladly accept XMR for payments, take a look at our list of merchants currently accepting Monero.
```
commerciants?
s/XMR for payments, take/XMR for payments; take/

erciccione replied:
Replaced `commerciants` (italianized word) with `entities`.

SarangNoether commented on _i18n/en.yml:
```diff
@@ -54,36 +54,33 @@ index:
+  guides_para: >
+    The community has put together a vast amount of resources and documentation. Users can find useful info and guides about common configurations and 
+    tips in the 'User Guides' section. Developers can find everything they need to build a service based on Monero or simply communicate with the network 
+    in the 'Developer Guides'. The Library instead, offers pubblications and books downloadable for free, including the full 'Mastering Monero' and 'Zero to Monero' books.
```
s/Library instead, offers/Library offers/

SarangNoether commented on _i18n/en.yml:
```diff
@@ -54,36 +54,33 @@ index:
+  joincommunity: Join the Community
+  contributecommunity: "We need your skills: Contribute!"
+  contributecommunity_para: Monero is not simply a currency, it's a decentralized community and exists only thanks to the hundred of volunteers that dedicate their time to the project. Join us!
+  researchlab_para: The MRL (Monero Research Lab) is a very important part of Monero. Researches and crytpographer from all over the world meet to find new ways to improve Monero. Read their papers and join them!
```
s/Researches/Researchers/
s/crytpographer/cryptographers/

SamsungGalaxyPlayer commented:
@erciccione I get that your intent is to remove a lot of information, but it goes too far in my opinion. While the main goal should be to get people started with the wallet and other resources, people need to be convinced to use Monero over other cryptos or other things. I think three words/icons is a reasonable way to reduce content while still keeping *some* there. Else we can do things like an orange banner under the first card saying something like "Do you want to learn what Monero is? Click here ->". I think the proposed screen is missing too much of the information people are looking for. Does that make sense?

erciccione commented:
I don't agree with the fact that we need to convince people to use Monero, but we could add some more basic info to the first block.

I wouldn't know how without making the "What is Monero" button redundant. I would exclude a banner because it would be inconsistent with the rest of the website and we use banners only for warnings.

> A third idea: replace the text description under "a reasonably private digital currency" with those small three icons. Why read a paragraph when you can get more out of icons and 3 words.

This sounds like a good compromise, but i'm not  sure of how we would implemente that. Maybe we could just put the video in the center and under it three blocks with each the logo?

SamsungGalaxyPlayer commented:
Here's my super crude example

![image](https://user-images.githubusercontent.com/12520755/81315816-67b63800-9050-11ea-81aa-ac1af3864b0c.png)

I also think adding "reasonably" was a mistake, but maybe that's for another issue. It's too long and doesn't actually help convey the nuances it attempts to convey.

erciccione commented:
@SamsungGalaxyPlayer I prefer the first one, but i'm not sure if removing the text and only leaving the "buttons" would be beneficial. I like the idea of the bigger "learn more about Monero" button tho.

> I now think adding "reasonably" was a mistake, but maybe that's for another issue

Please open an issue about it so that can be discussed there.

erciccione commented:
What about making the button at the top bigger in the meantime?
![top](https://user-images.githubusercontent.com/28106476/82319948-bc539e80-99d2-11ea-9664-6a2cc7a56a31.png)

We could improve more that section in another PR if we want.

erciccione commented:
Current preview:

![Screen Shot 2020-05-23 at 13 29 39-fullpage](https://user-images.githubusercontent.com/28106476/82729661-8cb9d480-9cf9-11ea-9df4-92073cb9809a.png)

SamsungGalaxyPlayer commented:
I can't test right now, but I approve of the screenshot.

erciccione commented:
Rebased

erciccione commented:
Could somebody review this PR? so if people lik it we can go ahead and merge it.

erciccione commented:
Rebased and pushed to trigger netlify's preview.

SarangNoether commented:
s/pubblications/publications

SarangNoether commented:
s/crytpographers/cryptographers

00-matt commented on _i18n/en.yml:
```diff
@@ -54,36 +54,33 @@ index:
-  monero_is_fungible_para2: fungible
-  monero_is_fungible_para3: because it is private by default. Units of Monero cannot be blacklisted by vendors or exchanges due to their association in previous transactions.
+  choosewallet: Choose a Wallet
+  choosewallet_para: To use Monero, the first thing you are going to need is a wallet. Visit our Downloads page and get the right wallet for you. The Monero wallets are available for a variety of platforms and contain everything you need to use Monero immediately and, if you want, run your own node.
```
Should we mention nodes yet? New users might not know what they are.

erciccione replied:
Yeah, makes sense. I will just remove everything after `immediately`.

00-matt commented on _i18n/en.yml:
```diff
@@ -54,36 +54,33 @@ index:
+  getcoins: Get Some Coins
+  getcoins_para: After you install a wallet, you need to get some Monero. There are multiple ways to acquire some coins to spend, like mining or working in exchange for Monero, but the easiest way is to use an exchange and convert your fiat money into XMR. many exchanges, centralized and decentralized, list Monero (XMR).
+  exchanges: Exchanges
+  useit: "It's a currency: Use it!"
```
Some of the headings are in title case, e.g. "Choose a Wallet", "Get Some Coins", but some (like this one) aren't. Not sure which style looks better but I thnk it's best to be consistent.

00-matt commented on _i18n/en.yml:
```diff
@@ -54,36 +54,33 @@ index:
+  merchants: Merchants
+  answers: Find answers to your questions
+  faq: FAQ
+  faq_para: We've heard a lot of questions over the years and have compiled, for your convenience, a thorough and varied FAQ. Don't worry, if your questions are not on here, you can always ask the community.
```
A bit clunky to read. Maybe something more like:

> We have compiled a list of frequently asked questions, and have a large community with members ready to answer any questions you cannot find the answer to.

erciccione replied:
On a second look i find myself not 100% convinced about your suggestion (no problem with changing my version tho). I would avoid repeating both 'questions' and 'answer' in such short sentence.

erciccione replied:
We agreed to just remove `, for your convenience,` from the original sentence.

00-matt commented on _i18n/en.yml:
```diff
@@ -54,36 +54,33 @@ index:
+  guides_para: >
+    The community has put together a vast amount of resources and documentation. Users can find useful info and guides about common configurations and 
+    tips in the 'User Guides' section. Developers can find everything they need to build a service based on Monero or simply communicate with the network 
+    in the 'Developer Guides'. The Library offers pubblications and books downloadable for free, including the full 'Mastering Monero' and 'Zero to Monero' books.
```
publications

00-matt commented on _i18n/en.yml:
```diff
@@ -54,36 +54,33 @@ index:
+  joincommunity: Join the Community
+  contributecommunity: "We need your skills: Contribute!"
+  contributecommunity_para: Monero is not simply a currency, it's a decentralized community and exists only thanks to the hundred of volunteers that dedicate their time to the project. Join us!
+  researchlab_para: The MRL (Monero Research Lab) is a very important part of Monero. Researchers and crytpographers from all over the world meet to find new ways to improve Monero. Read their papers and join them!
```
cryptographers

00-matt commented on index.html:
```diff
@@ -4,180 +4,145 @@
+            <div class="row middle-xs">
+                <div class="col-lg-7 col-md-7 col-xs-12 main-video">
+                    <div class="monero-video">
+                        <!--iframe width="560" height="315" src="https://www.youtube.com/embed/TZi9xx6aiuY" frameborder="0" allowfullscreen></iframe-->
```
Is it worth just deleting this comment? We can always just copy it back out of the Git history.

erciccione replied:
IIRC it was put there as a backup in case we had problems with the hosted video. I would just leae it there, since it doesn't really bother and we have a quick reference to the video on Youtube. I wouldn't rely on the git history, mostly because we would need to remember that once we had it and i think tha won't happen :)

00-matt commented on index.html:
```diff
@@ -4,180 +4,145 @@
-        <!-- END 3 ONE-THIRD BLOCKS ROW -->
+            <div class="row middle-xs info-block-row">
+                <div class="col-lg-6 col-md-6 col-sm-4 col-xs-4 center-xs first-md last-sm last-xs">
+                    <img src="/img/mrl-logo.png" alt="Monero Research Lab" class="main-icon">
```
Can we get a higher res MRL logo? With my browser maximised, it gets scaled up slightly and looks very blurry.

erciccione replied:
Fluffypony just sent me an SVG version of the logo, i will replace the png with this new version in the next push :)

erciccione commented:
- Distanciate text from buttons
- Replaced png logo with svg version
- Change 'Improving Monero' button to 'Contributing'
- Uniform all headings to use title case
- Added text suggestions and fixes

erciccione commented:
The preview is updated: https://github.com/monero-project/monero-site/pull/948#issuecomment-647147171

netrik182 commented:
My suggestions as a non-native english speaker:

> Don't worry, if your questions are not on here, you can always ask the community.

Change to:

> Don't worry, if your questions are not **in there**, you can always ask the community.

Also:

> The Library offers publications and books ~~downloadable~~ for free

Change to:

> The Library offers **downloadable** publications and books for free

And finally, the button `DEVELOPER GUIDES` is not wide enough to this text:

![image](https://user-images.githubusercontent.com/30935310/85594769-bc7d3480-b637-11ea-9d90-c39ae3ab36f1.png)

00-matt commented:
@netrik182 both versions of both sentences sound fine to me :+1: 

Also what browser are you using? The text fits in the button fine for me at any screensize on Firefox 77.

netrik182 commented:
> Also what browser are you using?

FF76, might be some strict settings from addons then. Don't worry.

00-matt commented:
Could be something to do with the fonts you have installed

erciccione commented:
Added @netrik182's suggestions and removed 'Moneropedia' from en.yml, since we don't want it translated.

We could shorten 'developer guides' into 'dev guides', but the issue would probably remainfor the other languages. I think this is an edge case, but i'm open to suggestions to avoid the issue.

netrik182 commented:
> but the issue would probably remain for the other languages.

Yeah, I think it's fine as is, since my strict setup is responsible for that, really.

In pt-br, for example, this happens some times too. But the build preview will help a lot in the sense of identifying the issues.
